contrib/schemas/sensor.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Can you please add ``README.md`` to ``contrib/schema/`` directory which describes what it contains and how to (re)generate those files using ``st2-generate-schemas``?

And let's please also add a note which says those files are auto-generated and should not be edited manually.

Reply:
Added README. JSON files do not allow comments so the note to not edit manually is in the README.